scripts/build-aci

```diff
@@ -26,79 +26,39 @@ if [ -z "$VERSION" ] ; then
     exit 1
 fi
 
-mkdir -p $IMAGEDIR/rootfs
-if [ ! -d $IMAGEDIR/rootfs -o ! -x $IMAGEDIR/rootfs ]; then
-    echo "$IMAGEDIR/rootfs is not a writeable directory"
-    exit 1
-fi
+acbuild --debug begin
 
-if [ -n "$(ls -A $IMAGEDIR/rootfs)" ]; then
-    echo "$IMAGEDIR/rootfs is not empty"
-    exit 1
-fi
+cat <<DF > hosts
```

Comment:
Since apparently I can't just think of everything in one review: let me be even more pedantic and tell you to use an actual mktemp, so we don't accidentally clobber someone's `./hosts` if it happens to exist.